pkg/cmd/multi-operator-manager/test/apply-configuration/apply_configuration_command.go

```diff
@@ -82,8 +84,14 @@ func (f *TestApplyConfigurationFlags) Validate() error {
 	if len(f.TestDirectory) == 0 {
 		return fmt.Errorf("--test-dir is required")
 	}
-	if len(f.OutputDirectory) == 0 {
-		return fmt.Errorf("--output-dir is required")
+	if f.ReplaceExpectedDir {
+		if len(f.OutputDirectory) > 0 {
+			return fmt.Errorf("--output-dir cannot be specified with --replace-expected-output-dir")
+		}
+	} else {
```

Comment:
Where is `ReplaceExpectedDir` parsed, and how does it differ from `ReplaceExpectedOutput`? 

I don't see a `--replace-expected-output-dir` flag being bound in `BindFlags`.